tests/cephadm/test_bootstrap.py

```diff
@@ -269,6 +269,7 @@ def copy_ceph_configuration_files(cls, ceph_conf_args):
           output-keyring : "/root/ceph/ceph.client.admin.keyring"
           output-config : "/root/ceph/ceph.conf"
           output-pub-ssh-key : "/root/ceph/ceph.pub"
+          ssh-public-key : "/root/ceph/ceph.pub"
```

Comment:
I think this would fail in case `--ssh-public-key` is not included in bootstrap option.
Since `ssh-public-key` will have no value and copying that will lead into failure.

please try negative testing here.

Reply:
If possible, please add a verification of `ssh-public-key`.

Reply:
Added the verification as discussed yesterday. Added a new method **verify_ceph_custom_public_ssh_key(cls, ssh_key_path)** for the same. Please review and let me know if this is okay.

Reply:
Added validation for ssh public key and private key.